docs/patterns.rst

```diff
@@ -54,7 +54,7 @@ setting other values, e.g.::
 
         @property
         def LANGUAGES(self):
-            return Configuration.LANGUAGES + (('tlh', 'Klingon'),)
+            return Configuration.LANGUAGES + [('tlh', 'Klingon'),]
```

Comment:
Yes, I think it would be better.

Apart from that: you don't need the tuple-trailing-comma in your original patch.